### PR TITLE
feat: implement `sortedSetPutElement`

### PR DESCRIPTION
**Pull request by malandis**
- refactor: rename `SortedSetPut` response include `Elements` suffix
- feat: add `sortedSetPutElement` method, request, and response

TODO for future PR: tests

### CODE REVIEW

File: momento/sorted_set_test.go [hunk 1]
```diff
@@ -699,6 +722,44 @@ var _ = Describe("SortedSet", func() {
 		})
 	})
 
+	Describe(`SortedSetPutElement`, func() {
+		// TODO
```
Comment (pgautier404) [inline momento/sorted_set_test.go:726]: Is this testing coming in a followup PR?

Comment (pgautier404) [reply momento/sorted_set_test.go:726]: Ah, just saw your note in the description. All good.

Comment (malandis) [reply momento/sorted_set_test.go:726]: Yes

**Review by pgautier404 — APPROVED** [review]
LGTM!

**Review by cprice404 — APPROVED** [review]

**Pull request merged**